.ci/util.sh

```diff
@@ -12,6 +12,12 @@ function checkForVariable() {
     echo "Error: Define $1 environment variable"
     exit 1
   fi
+
+  VAR_VALUE="${!VAR_NAME}"
+  if [ -z "$VAR_VALUE" ]; then
```

Comment:
Hm `if [ ! -v "$VAR_NAME" ];` , can you receched docs for it, is way we tried to do this, it use to work. Is it a nuances of GitHub initialization of variable ?

Reply:
`-v` only checks if a variable is initialized, it does not care what value is given to the variable. I put documentation reference in PR description: https://www.gnu.org/software/bash/manual/html_node/Bash-Conditional-Expressions.html

Github actions uses an empty string if a secret is not set:
https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow
> If a secret has not been set, the return value of an expression referencing the secret (such as ${{ secrets.SuperSecret }} in the example) will be an empty string.

So the 
```yml
env:
   GPG_KEY: ${{ secrets.GPG_KEY }}
```
would be equal to
```sh
export GPG_KEY=""
```

The `if [ ! -v "$VAR_NAME" ];` statement would see that the variable is initialized but does not see that it is an empty string. Hence, we need an extra check.

Reply:
Ok, please make message `Set not empty value to ...`

Reply:
I propose we keep old `-v`check and add new `-z` check, with an error message that makes it very clear why check is failing. Github actions can be tricky to debug, any extra details will prove very helpful.

Reply:
Done.

Reply:
Yes, we keep both